kommitters/calendso
# Fixed stripe integration and delete event from google calendar

## What does this PR do?

- Fixed bug when you trying to book an event with payment, this event was booked and saved in google calendar even if it showed an error. So if you want to book an event with payment, it will show an error and the reservation will not be reserved and will not take availability.

- Fixed bug when you have unpaid or unconfirmed events bookings that need payment and you disconnect the payment method in installed apps, all those bookings are going to be canceled automatically.

Fixes #2170 

## Type of change




## How should this be tested?



Test for the first bug.
1. Go to apps/installed.
2. Connect a Stripe account.
3. Create a priced event type.
4. Remove Stripe account.
5. Try to book paid event type.

Test for the second bug.
1. Go to bookings/upcoming and see the unpaid or unconfirmed bookings.
2. Go to apps/installed and disconnect Stripe account.
3. Go to bookings/canceled and you are going to see all unpaid and unconfirmed bookings.
